templates/partial/_channel-map.html

```diff
@@ -32,6 +32,10 @@
     </div>
   </div>
 </div>
+<div class="juju-notification">
+  <i class="p-icon--warning" style="margin-right: 0.25rem;"></i>
+  You will need Juju 2.9 to be able to run this command. <a href="https://juju.is/docs/olm/upgrading">Learn how to upgrade to Juju 2.9</a>.
```

Comment:
This should be a relative link

Reply:
@anthonydillon this page is on charmhub.io - the link is a different domain